src/Tile_Maps.md

```diff
@@ -33,15 +33,53 @@ Bit 3    Tile VRAM Bank number      (0=Bank 0, 1=Bank 1)
 Bit 2-0  Background Palette number  (BGP0-7)
 ```
 
-When Bit 7 is set, the corresponding BG tile will have priority above
-all OBJs (regardless of the priority bits in OAM memory). There's also
-a Master Priority flag in LCDC register Bit 0 which overrides all other
-priority bits when cleared.
-
 Note that, if the map entry at `0:9800` is tile \$2A, the attribute at
 `1:9800` doesn't define properties for ALL tiles \$2A on-screen, but only
 the one at `0:9800`!
 
+### BG-to-OBJ Priority in CGB Mode
+
+In CGB Mode, the priority between the BG (and window) layer and the OBJ layer is declared in three different places:
+- [BG Map Attribute bit 7](<#BG Map Attributes (CGB Mode only)>)
+- [LCDC bit 0](<#LCDC.0 — BG and Window enable/priority>)
+- [OAM Attributes bit 7](<#Byte 3 — Attributes/Flags>)
+
+We can infer the following rules from the table below:
+* If the BG color index is 0, the OBJ will always have priority;
+* Otherwise, if LCDC bit 0 is clear, the OBJ will always have priority;
+* Otherwise, if both the BG Attributes and the OAM Attributes have bit 7 clear, the OBJ will have priority;
+* Otherwise, BG will have priority.
+
+::: tip
+
+BG Map attributes bit 7 is used to override OAM attributes bit 7
+
+:::
+
+The following table shows the relations between the 3 flags:
+
+LCDC bit 0 | OAM attr bit 7 | BG attr bit 7 | Priority
+:---------:|:--------------:|:-------------:|---------
+0          | 0              | 0             | OBJ
+0          | 0              | 1             | OBJ
+0          | 1              | 0             | OBJ
+0          | 1              | 1             | OBJ
+1          | 0              | 0             | OBJ
+1          | 0              | 1             | BG color 1–3, otherwise OBJ
+1          | 1              | 0             | BG color 1–3, otherwise OBJ
+1          | 1              | 1             | BG color 1–3, otherwise OBJ
```

Comment:
```suggestion
0          | 1              | 0             | OBJ
0          | 1              | 1             | OBJ
0          | 0              | 0             | OBJ
0          | 0              | 1             | OBJ
1          | 1              | 0             | OBJ
1          | 1              | 1             | BG color 1–3, otherwise OBJ
1          | 0              | 0             | BG color 1–3, otherwise OBJ
1          | 0              | 1             | BG color 1–3, otherwise OBJ
```
Looks like we forgot that OBJ attributes bit 7 is inverted... right?

Reply:
The original is the actual value of the bit, not sure if you want to invert it but I think it will more confusing seeing in this table inverted values for this bit (there is also a warning below stating that OBJ get priority upon clear and not set).

Reply:
When the bit is 0, the OBJ loses priority, so  `1 0 0` shows BG, not the OBJ. The original appears not to be `[$FExx] >> 7`.

Reply:
No, it is correct, I'm just being confused.